CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
+## 5.4.2 - 2023-09-22
+### Fixed
+- Lost contact groups
```

Comment:
```suggestion
- Lost contact groups

```

I think this is need for the app store changelog parser to understand the changelog file

Reply:
re https://github.com/nextcloud/contacts/pull/3605#issuecomment-1730230531